ausroller.py

```diff
@@ -1,244 +1,19 @@
-#!/usr/bin/env python2
-# encoding: utf-8
+#!/usr/bin/python
```

Comment:
Good point. But the hashbang line is superflous in core.py and main() too :-D 
Will fix that